### DISCUSSION TITLE
a `shortest init` command ?

### DISCUSSION THREAD

**Discussion by amk-dev** [antiwork/shortest#174] [Ideas]
a new `shortest init` command ?

similar to `tailwind init`

this could take care of multiple steps users have to do,
1. creating a `shortest.config.ts` file
2. adding `.shortest` to `.gitignore`
3. asking the user to install the browsers they want to test with

or anything else that needs setting up 

users should be done with

```
npm install @antiwork/shortest
npx shortest init
```

**Comment by slavingia** [comment]
Sure!

**Reply by amk-dev** [reply]
great. will get to implementing this

**Comment by rmarescu** [comment]
This was released in [v0.4.3](https://github.com/anti-work/shortest/releases/tag/v0.4.3)!
